pkg/trace/obfuscate/sql_tokenizer.go

```diff
@@ -223,6 +223,9 @@ func (tkn *SQLTokenizer) Scan() (TokenKind, []byte) {
 				tkn.advance()
 				return ColonCast, []byte("::")
 			}
+			if unicode.IsSpace(tkn.lastChar) {
```

Comment:
Is it ever possible for the space to be missing after `autovacuum:`? I assume the answer is negative but asking just to be 100% sure.

Reply:
yeah, I can't find any documentation that contradicts this